app/controllers/api/v1/workflows_controller.rb

```diff
@@ -72,7 +72,9 @@ def extract_strings(tasks)
 
   def add_relation(resource, relation, value)
     if relation == :retired_subjects && value.is_a?(Array)
+      resource.save!
```

Comment:
Yes, but I don't feel confident enough in the whole JsonApiController stuff that I dare say whether there's any way there aren't any unsaved changes. If you do, then cool I'll remove this.